search-sentiments-map/public/scripts/map-script.js

```diff
@@ -141,10 +141,10 @@ function loadCountryData() {
   let dataVariableMax = Number.MIN_VALUE;  // Smallest positive number.
   let dataVariableMin = Number.MAX_VALUE;
   // Countries with the minimum and maximum scores for current topic. 
-  let countryMaxName = '';
-  let countryMaxId = '';
-  let countryMinName = '';
-  let countryMinId = '';
+  let maxScoreCountryName = '';
+  let maxScoreCountryId = '';
+  let minScoreCountryName = '';
+  let minScoreCountryId = '';
```

Comment:
Alphabetical order so the max (Id then name) + min (id then name)

Reply:
I think in a previous comment similar to this one, you said that we wanted the countryNames together and the countryId's together? or is it different for this case?

Reply:
Changed it to follow Haylee's suggestion. If we change our mind on this let me know.